### DISCUSSION TITLE
How to add Losslesscut to Send to?

### DISCUSSION THREAD

**Discussion by linegrinder** [mifi/lossless-cut#1857] [Support]
Hi there,
I have tried setting up losslesscut with Send to so I may right-click on a video in my explorer and directly send it to losslesscut. I created a shortcut, added -v -i after it and added it to the shell:sendto directory. However this doesn't seem to work properly as it only opens losslesscut but doesn't directly use the selected video as input. Any help?

**Comment by mifi** [comment]
which OS are you on?
